### DISCUSSION TITLE
how can i use digger.js?

### DISCUSSION THREAD

**Discussion by LordFox2812** [PrismarineJS/mineflayer#1891] [Q&A] [answered]
???

**Comment by amoraschi** [comment]
Read the Tutorial, mark my words 👍

**Reply by LordFox2812** [reply]
i put everything in but it doesn't work

**Reply by amoraschi** [reply]
Define: doesn't work

**Comment by rom1504** [comment, marked as answer]
this is not a good question.
If you want answers, try harder, you can look at https://github.com/PrismarineJS/mineflayer/blob/master/.github/ISSUE_TEMPLATE/bug_report.md for inspiration
